fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -432,10 +437,56 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            log_Msg "---> No distortion correction"
+
+            log_Msg "---> FAKE WarpField"
+            ### faked warp image with three frames
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 ${WD}/WarpField.nii.gz
+            ${FSLDIR}/bin/fslmerge -t ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz
+
+            log_Msg "---> FAKE Jacobian Volume"
+            ###Jacobian Volume FAKED for Regular Fieldmaps (all ones) ###
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 -add 1 -bin ${WD}/Jacobian.nii.gz
+
+            log_Msg "---> FAKE spline interpolated image of scout"
+            # create a spline interpolated image of scout (distortion corrected in same space)
+            ${FSLDIR}/bin/imcp ${ScoutInputName} ${WD}/${ScoutInputFile}_undistorted
+
+            log_Msg "---> REORIENT scout to standard MNI152 template"
+            # use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images (MNI152)
+            fslreorient2std ${WD}/${ScoutInputFile}_undistorted.nii.gz ${WD}/${ScoutInputFile}_undistorted_fMRI2str.nii.gz
```

Comment:
Another `fslreorient2std`.  Is this necessary?  Not sure of the possible unintended downstream consequences of this particular reorienting...

Reply:
I agree, this is a mess right now. Let's do both BOLD and Scout reorient in a single place only if DC method is NONE. The key question is where. I think It needs to be before motion correction (as the reference used can be another BOLD that already had `fslreorient2std` done.

Reply:
I moved the Scout reorient to be done with BOLD reorient in Slice Time Correction PR.